jobs/scripts/parallel-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+sed -i 's/    mr.developer/    mr.developer\ngit-clone-depth = 10/' core.cfg
+
+pip install -r requirements.txt
+buildout -c py3.cfg
+
+xvfb-run -a --server-args='-screen 0 1920x1200x24' bin/test -j 6 --xml -vvv
```

Comment:
Missing `--all`.

We should not use any verbosity flags as `zope.testrunner` has gone a bit silly on that.

https://github.com/zopefoundation/zope.testrunner/blob/cfb6aab7aefe6d2ffb6241b10847a636c631f6d4/src/zope/testrunner/runner.py#L675-L682

We do not want it to run via the `KeepaliveSubprocessResult` but via `DeferredSubprocessResult` as otherwise the output is weird to the point of being able to mask failures.

So `bin/test --all --xml -j 6`. Or if the weight parametre is exposed as a `{variable}` here, that'd be even better than having two places to keep track of this.

Reply:
Done